src/nunavut/lang/properties.yaml

```diff
@@ -312,7 +312,9 @@ nunavut.lang.cpp:
         enable_serialization_asserts: false
         enable_override_variable_array_capacity: false
         std: c++14
-        variable_array_type: "std::vector<{TYPE},std::allocator<{TYPE}>>"
+        # Provide non-empty values to override the type used for variable-length arrays in C++ types.
+        variable_array_type_template: ""
```

Comment:
The internal logic is now "if user doesn't override then use our built-in VLA type."